### PR TITLE
Add an how-to about dependencies versions check.

### PR DESCRIPTION
**Pull request by bguedas**

### CODE REVIEW

File: SDK6UserGuide/howtos/checkDependenciesVersions.rst [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+
+Some dependencies are automatically included by the MicroEJ Gradle plugin to compile the project with JDT::
+
+    - org.eclipse.jdt:org.eclipse.jdt.compiler.apt:1.3.1000
+    - org.eclipse.jdt:org.eclipse.jdt.core:3.22.    
+    
+These dependencies are not relevant for  the report. To remove them, you can add the following snippet to the build file::
+
+    tasks.withType<DependencyUpdatesTask> {
+        configurations.jdtCompilerClasspath {
```
Comment (denisbedin) [inline SDK6UserGuide/howtos/checkDependenciesVersions.rst:35]:
```suggestion
        configurations.all {

```

**Review by denisbedin — CHANGES_REQUESTED** [review]

File: SDK6UserGuide/howtos/checkDependenciesVersions.rst [hunk 2]
```diff
@@ -0,0 +1,141 @@
+.. _sdk_6_check_dependency_versions:
+
+How To Check Dependencies Updates
+=================================
+
+To check if Gradle dependencies are up-to-date, you can use the `Gradle Versions Plugin <https://github.com/ben-manes/gradle-versions-plugin>`__.
+This Gradle plugin lists all the dependencies declared in the ``build.grate.kts`` file, and tells whether they are up-to-date or if a new version is available.
```
Comment (tdelhomenie-microej) [inline SDK6UserGuide/howtos/checkDependenciesVersions.rst:7]:
```suggestion
This Gradle plugin lists all the dependencies declared in the ``build.gradle.kts`` file, and tells whether they are up-to-date or if a new version is available.

```

File: SDK6UserGuide/howtos/checkDependenciesVersions.rst [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+   .. tab:: Command Line Interface
+
+      From the command line interface::
+      
+          $ ./gradlew dependenciesUpdates
+
+A simple text report is printed in the console and generated in ``build/dependenciesUpdates/report.txt``.
+Report is configurable in the plugin. Refer to the `Report Format <https://github.com/ben-manes/gradle-versions-plugin/tree/v0.51.0?tab=readme-ov-file#report-format>`__ section of the plugin README.
+
+For instance, here is the report generated on `Example-Java-Widget 8.1.0 <https://github.com/MicroEJ/Example-Java-Widget/tree/8.1.0>`__::
```
Comment (tdelhomenie-microej) [inline SDK6UserGuide/howtos/checkDependenciesVersions.rst:89]:
```suggestion
Here is an example of report::

```

File: SDK6UserGuide/howtos/checkDependenciesVersions.rst [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+    The example uses MicroEJ Application plugin ``0.15.0`` which does not provide the ``dependenciesVersionsCheckEnabled`` option.
+    Moreover, this plugin version fetches extra dependencies at rutime that can be filtered::
+        
+        tasks.withType<DependencyUpdatesTask> {
+	        configurations.all {
+		        exclude(group = "org.eclipse.jdt")
+		        exclude(group = "org.openjdk.nashorn")
+		        exclude(group = "org.apache.ant")
+	        }
+        }
```
Comment (tdelhomenie-microej) [inline SDK6UserGuide/howtos/checkDependenciesVersions.rst:135]: I would remove this note completely.

Comment (bguedas) [reply SDK6UserGuide/howtos/checkDependenciesVersions.rst]: Done

**Review by tdelhomenie-microej — APPROVED** [review]

File: SDK6UserGuide/howtos/checkDependenciesVersions.rst [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+Some dependencies are automatically included by the MicroEJ Gradle plugin to compile the project with JDT::
+
+    - org.eclipse.jdt:org.eclipse.jdt.compiler.apt:1.3.1000
+    - org.eclipse.jdt:org.eclipse.jdt.core:3.22.    
+    
+These dependencies are not relevant for  the report. To remove them, you can add the following snippet to the build file::
+
+    tasks.withType<DependencyUpdatesTask> {
+        configurations.all {
+
```
Comment (denisbedin) [inline SDK6UserGuide/howtos/checkDependenciesVersions.rst:37]:
```suggestion

```

**Review by denisbedin — APPROVED** [review]

**Pull request merged**